### PR TITLE
Added 1.0.2 changes to changelog 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added 1.0.2 changes to changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,182 @@
 # Changelog
 
+## 1.0.2 
+
+### Additions and Improvements
+
+- Removed DB init when using `public-key` subcommand [\#1049](https://github.com/PegaSysEng/pantheon/pull/1049) 
+- Added `public-key address export` subcommand [\#888](https://github.com/PegaSysEng/pantheon/pull/888)
```
Comment (NicolasMassart) [inline CHANGELOG.md]: #888 was already in 1.0.1, that's the one we added and was the reason for the 1.0.1

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,182 @@
...
+   - Updated [documentation contribution guidelines](https://github.com/PegaSysEng/pantheon/blob/master/DOC-STYLE-GUIDE.md) 
+   - Added [`admin_removePeer`](https://docs.pantheon.pegasys.tech/en/latest/Reference/JSON-RPC-API-Methods/#admin_removepeer) 
+   - Updated [tutorials](https://docs.pantheon.pegasys.tech/en/latest/Tutorials/Create-Private-Clique-Network/) for printing of enode on startup 
+   - Added [`txpool_pantheonTransactions`](https://docs.pantheon.pegasys.tech/en/latest/Reference/JSON-RPC-API-Methods/#txpool_pantheontransactions) 
+   - Added [Transaction Pool content](https://docs.pantheon.pegasys.tech/en/latest/Using-Pantheon/Transactions/Transaction-Pool/) 
+   - Added [`tx-pool-max-size` CLI option](https://docs.pantheon.pegasys.tech/en/latest/Reference/Pantheon-CLI-Syntax/#tx-pool-max-size) 
+   - Updated [developer build instructions to use installDist](https://github.com/PegaSysEng/pantheon/blob/master/docs/development/running-developer-builds.md) 
+   - Added [Azure one click tutorial](https://docs.pantheon.pegasys.tech/en/latest/Tutorials/Azure/Azure-Private-Network-Quickstart/)
```
Comment (NicolasMassart) [inline CHANGELOG.md]: One click is just one part of the addition

```suggestion
   - Added [Azure quickstart tutorial](https://docs.pantheon.pegasys.tech/en/latest/Tutorials/Azure/Azure-Private-Network-Quickstart/) 
```

File: CHANGELOG.md (hunk 1 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Fixed

**Pull request merged**